internal/dbmodel/group.go

```diff
@@ -6,15 +6,17 @@ import (
 	"time"
 
 	"github.com/juju/names/v5"
-	"gorm.io/gorm"
 
 	apiparams "github.com/canonical/jimm/v3/pkg/api/params"
 	jimmnames "github.com/canonical/jimm/v3/pkg/names"
 )
 
 // A GroupEntry holds information about a user group.
 type GroupEntry struct {
-	gorm.Model
+	// Note this doesn't use the standard gorm.Model to avoid soft-deletes.
+	ID        uint `gorm:"primarykey"`
```

Comment:
i could be wrong, but when creating `GroupEntry` this ID is always set to 0. Right?
Or the form struct tag does something to it?

Reply:
The default value is 0 but gorm will assign the database ID to it.

Reply:
yes, i was worried we were using that ID for something in the tests. But I've checked and it is alright